lib/Adapters/TestAdapter.js

```diff
@@ -10,6 +10,7 @@ var networkClient = require('./../NetworkClientFactory').getNetworkClient();
 var url=require('url');
 
```

Comment:
1) support multiple different recording of same entry, based on override flag.
2) fixed the naming convention of recorded stream output directory to avoid using spaces in name.
3) added timer to limit recording time based on configuration.